pyproject.toml

```diff
@@ -43,6 +43,14 @@ opentelemetry-exporter-jaeger-thrift = "^1.13.0"
 optional = true
 dependencies = { sentry-sdk = "^1.11.0" }
 
+[tool.poetry.group.flask_api]
```

Comment:
Pedantic, but please keep alphabetical order (please move this below `fast_api` and both above `open_telemetry`)